nla-job/README.md

```diff
@@ -0,0 +1,3 @@
+# nla-job
+
+Runs a query in Athena to obtain information (web title, url, pageviews) for content of types 'type/article' and 'tone/minutebyminute' for that day, and stores the results in Amazon S3. 
```

Comment:
in AWS Athena.
So people don't stop to wonder is Athena is a Guardian thing or a AWS product.

Reply:
Good point!